networkx/algorithms/bipartite/tests/test_generators.py

```diff
@@ -141,19 +141,19 @@ def test_alternating_havel_hakimi_graph(self):
         
         bseq=[2,2,2,2,2,2]
         G=alternating_havel_hakimi_graph(aseq,bseq)
-        assert_equal(sorted(G.degree().values()),
+        assert_equal(sorted(dict(G.degree()).values()),
                      [2, 2, 2, 2, 2, 2, 3, 3, 3, 3])
 
         aseq=[2,2,2,2,2,2]
         bseq=[3,3,3,3]
         G=alternating_havel_hakimi_graph(aseq,bseq)
-        assert_equal(sorted(G.degree().values()),
+        assert_equal(sorted(dict(G.degree()).values()),
                      [2, 2, 2, 2, 2, 2, 3, 3, 3, 3])
 
         aseq=[2,2,2,1,1,1]
         bseq=[3,3,3]
         G=alternating_havel_hakimi_graph(aseq,bseq)
-        assert_equal(sorted(G.degree().values()),
+        assert_equal(sorted(dict(G.degree()).values()),
```

Comment:
This could be:  [not sure it matters for tests... but its a good idiom to propagate in the code base.

    assert_equal(sorted( d for n,d in G.degree()),